src/bundle.c

```diff
@@ -729,6 +729,7 @@ gboolean check_bundle(const gchar *bundlename, RaucBundle **bundle, gboolean ver
 				error,
 				ierror,
 				"Failed to seek to end of bundle: ");
+		res = FALSE;
```

Comment:
The surrounding if-clause just proved that `res` is already `FALSE`, so why set it again here?

Reply:
Indeed, I'll amend.